internal/cmd/ansible-operator/run/cmd.go

```diff
@@ -183,11 +188,6 @@ func run(cmd *cobra.Command, f *flags.Flags) {
 		}, w.Blacklist)
 	}
 
-	err = mgr.AddHealthzCheck("ping", healthz.Ping)
```

Comment:
Removing this is a breaking change

Reply:
Added a todo and a msg in the fragment to use the endpoints provided instead of it.